util/libssl.num

```diff
@@ -514,3 +514,4 @@ SSL_set_quic_transport_version          20012	1_1_1i	EXIST::FUNCTION:QUIC
 SSL_get_peer_quic_transport_version     20013	1_1_1i	EXIST::FUNCTION:QUIC
 SSL_get_quic_transport_version          20014	1_1_1i	EXIST::FUNCTION:QUIC
 SSL_set_quic_early_data_enabled         20015	1_1_1j	EXIST::FUNCTION:QUIC
+SSL_new_session_ticket                  20016	1_1_1k	EXIST::FUNCTION:
```

Comment:
This is ok if we assume that this function won't eventually be added to 1.1.1. Do we think it will be?

Reply:
The new API implements a new feature, so it would take an OTC vote to approve an exception policy in order for it to make it to 1.1.1.  I don't expect that to happen (though I can't definitively rule it out).

Reply:
That's fair.